spec/support/features/idv_helper.rb

```diff
@@ -88,12 +88,12 @@ def choose_idv_otp_delivery_method_voice
     click_idv_send_security_code
   end
 
-  def visit_idp_from_sp_with_ial2(sp, **extra)
+  def visit_idp_from_sp_with_ial2(sp, client_id_override: nil, **extra)
     if sp == :saml
       visit_idp_from_saml_sp_with_ial2
     elsif sp == :oidc
       @state = SecureRandom.hex
-      @client_id = sp_oidc_issuer
+      @client_id = client_id_override || sp_oidc_issuer
```

Comment:
I think I am going to split this out into another PR later.

I think this was a good idea, but IdvHelper ends up just passing strings around, not instantiating a real ServiceProvider, so the hardcoded redirect and nonce don't work. I'd like to make it operate on an actual ServiceProvider object, to more easily allow passing in an override.

Reply:
Filed as LG-12303